src/components/item/iterable.js

```diff
@@ -22,12 +28,19 @@ class ItemIterable extends React.PureComponent {
       'item': true,
       'drop-target': !this.props.isDisabled,
       'active': this.props.isSelected,
-      'over': this.props.isOver && this.props.canDrop,
+      'over': this.props.isOver &&
```

Comment:
Let's simplify this: I think we can skip the `canDrop` prop (I don't think we really use it for rendering purposes?) and in the DropTargetCollect function set `isOver` to true only if `isOver` and `canDrop` are both true.